client/RoomClient.js

```diff
@@ -843,13 +843,70 @@ export default class RoomClient extends EventTarget
 						break;
 					}
 
-					case 'initState':
+					case 'initState': {
+						console.log('Room.js, initState:', notification);
+						break;
+					}
 					case 'updateState':
 					{
+						console.log('Room.js, updateState:', notification);
+						this.dispatchEvent(new MessageEvent(notification.method, {
+							data: notification.data,
+						}));
+						break;
+					}
+
+					case 'setState':
+					{
+						console.log('Room.js, setState:', notification)
+						this.dispatchEvent(new MessageEvent(notification.method, {
+							data: notification.data,
+						}));
+						break;
+					}
+
+					case 'getFile':
+					{
+						console.log('Room.js, getFile:', notification)
+						this.dispatchEvent(new MessageEvent(notification.method, {
+							data: notification.data,
+						}));
+						break;
+					}
+
+					case 'edit':
+					{
+						console.log('Room.js, edit:', notification)
+						this.dispatchEvent(new MessageEvent(notification.method, {
+							data: notification.data,
+						}));
+						break;
+					}
+
+					case 'getAllKeys':
+					{
+						console.log('Room.js, getAllKeys:', notification)
+						this.dispatchEvent(new MessageEvent(notification.method, {
+							data: notification.data,
+						}));
+						break;
+					}
+
+					case 'runCode':
+					{
+						console.log('Room.js, runCode:', notification)
 						this.dispatchEvent(new MessageEvent(notification.method, {
 							data: notification.data,
 						}));
+						break;
+					}
 
+					case 'uploadBinary':
```

Comment:
If this isn't used we should get rid of it.

Reply:
removed unused apis